neuvue_project/neuvue/settings.py

```diff
@@ -254,4 +254,5 @@
 
 # NEURD variables
 # should this be an env var?
-NEURD_LAMBDA_URL = "https://g9v90mw9u0.execute-api.us-east-1.amazonaws.com/production"
\ No newline at end of file
+NEURD_LAMBDA_URL = "https://g9v90mw9u0.execute-api.us-east-1.amazonaws.com/production"
+NEURD_LAMBDA_SECRET_ARN = ""
```

Comment:
Nitpicky thing but we can follow a similar pattern on how we do other secrets here if we want to be consistent:
```python
if "RDS_HOSTNAME" in os.environ and DEBUG is False:
    DATABASES = {
        "default": {
            "ENGINE": "django.db.backends.mysql",
            "NAME": os.environ["RDS_DB_NAME"],
            "USER": os.environ["RDS_USERNAME"],
            "PASSWORD": os.environ["RDS_PASSWORD"],
            "HOST": os.environ["RDS_HOSTNAME"],
            "PORT": os.environ["RDS_PORT"],
        }
```

Reply:
(i.e.) check environment variables directly and set a default value